Tests/Helpers/MailAddressNormalizerTests.cs

```diff
@@ -0,0 +1,86 @@
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using VocaDb.Model.Helpers;
+
+namespace VocaDb.Tests.Helpers {
+
+	/// <summary>
+	/// Code from https://github.com/iDoRecall/email-normalize/blob/0938e0a4710c6fc076c50dd42ea2886b2984e219/tests.js
+	/// </summary>
+	[TestClass]
+	public class MailAddressNormalizerTests {
+
+		private async Task<string> NormalizedAddress(string address, MailAddressNormalizerOptions options = MailAddressNormalizerOptions.None) {
+			return await MailAddressNormalizer.NormalizeAsync(address, options);
+		}
+
+		[TestMethod]
+		public async Task OnlySupportedDomains() {
+			Assert.AreEqual("a.b.c+tag@example.com", await NormalizedAddress("a.b.c+tag@example.com"));

```

Comment:
You could simplify the tests further by extracting TestNormalizedEmail method, taking expected and given email
```
public void TestNormalizedEmail(string expected, string given) { 
    Assert...
}
```